cli/cmd/check.go

```diff
@@ -126,10 +130,20 @@ func (o *checkOptions) validate() error {
 	if o.preInstallOnly && o.dataPlaneOnly {
 		return errors.New("--pre and --proxy flags are mutually exclusive")
 	}
+	if o.output != "basic" && o.output != "json" {
```

Comment:
mind using the `jsonOutput` constant defined in `root.go`? also instead of "basic", i think it's ok to just use `tableOutput`, to match the other commands.

Reply:
oh nice, I was not aware of that